todo_app/app.py

```diff
@@ -42,4 +57,14 @@ def reopen_item(id):
     return redirect(url_for('index'))
 
 if __name__ == '__main__':
-    app.run()
\ No newline at end of file
+    app.run()
+
+###
+#def create_app():
+#    app = Flask(__name__)
+#    # We specify the full path and remove the import for this config so
+#    # it loads the env variables when the app is created, rather than when this file is imported
+#    app.config.from_object('todo_app.flask_config.Config')
+#    # All the routes and setup code etc
```

Comment:
The idea is that you move lines 14 to 60 in here.
`poetry run flask run` will continue to work because it picks up the create_app function automatically, and the advantage is that the integration test can then control when the setup happens (i.e. it can create the app **after** setting the environment up) 

Note that you will need to update trello_items.py to work with the test. Currently it reads from the environment as soon as the file is imported. before the test has a chance to set test env vars. You'll want to move those lines 7 to 13 into functions

Reply:
Hi Hugh are you suggesting seperate functions for each or a general one, parameter based 
i.e "dev", Test" etc

Reply:
got over the hurdle...... work is continuing